README.md

```diff
@@ -59,21 +59,35 @@ oc edit -n etcd-backup cm/backup-config
 ```
 
 The following options are used:
-- `backup.subdir`: Sub directory on PVC. If it not exists it will be created.
-- `backup.dirname`: Dirname of singe backup. This is a string which run trough
+- `OCP_BACKUP_SUBDIR`: Sub directory on PVC. If it not exists it will be created.
+- `OCP_BACKUP_DIRNAME`: Dirname of singe backup. This is a string which run trough
 [`date`](https://man7.org/linux/man-pages/man1/date.1.html)
-- `backup.expiretype`:
+- `OCP_BACKUP_EXPIRE_TYPE`:
   - `days`: Keep backups newer than `backup.keepdays`.
   - `count`: Keep a number of backups. `backup.keepcount` is used to determine how much.
   - `never`: Dont expire backups, keep all of them.
-- `backup.keepdays`: Days to keep the backup. Only used if `backup.expiretype` is set to `days`
-- `backup.keepcount`: Number of backups to keep. Only used if `backup.expiretype` is set to `count`
+- `OCP_BACKUP_KEEP_DAYS`: Days to keep the backup. Only used if `backup.expiretype` is set to `days`
+- `OCP_BACKUP_KEEP_COUNT`: Number of backups to keep. Only used if `backup.expiretype` is set to `count`
+- `OCP_BACKUP_UMASK`: Umask used inside the script to set proper permission on written files.
```

Comment:
> to set proper permission on written files

to set restrictive permissions on the written files, as they might contain sensitive information.

Reply:
```suggestion
- `OCP_BACKUP_UMASK`: Umask used inside the script to set restrictive proper permission on written files, as they contain sensitive information.
```